client/src/components/main/features/access-block.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
```

Comment:
Since all of these are just static, can we have one component that renders all three?